yb-voyager/src/srcdb/postgres.go

```diff
@@ -175,8 +175,8 @@ func (pg *PostgreSQL) ExportSchema(exportDir string) {
 	pgdumpExtractSchema(pg.source, pg.getConnectionUri(), exportDir)
 }
 
-func (pg *PostgreSQL) ExportData(ctx context.Context, exportDir string, tableList []string, quitChan chan bool, exportDataStart chan bool) {
-	pgdumpExportDataOffline(ctx, pg.source, pg.getConnectionUri(), exportDir, tableList, quitChan, exportDataStart)
+func (pg *PostgreSQL) ExportData(ctx context.Context, exportDir string, tableList []string, quitChan chan bool, exportDataStart, exportSuccessChan chan bool) {
+	pgdumpExportDataOffline(ctx, pg.source, pg.getConnectionUri(), exportDir, tableList, quitChan, exportDataStart) // No need to use exportSuccessChan
```

Comment:
Let's have a uniform behaviour: `exportSuccessChan` is signalled whenever the export data tool (ora2pg/pg_dump) completes successfully.
That should allow us getting rid of explicit checks for the Oracle/MySQL DBType.